prisma/schema.prisma

```diff
@@ -283,9 +285,27 @@ model UserResponse {
 }
 
 model ResponseSubpart {
-  id              String        @id @default(auto()) @map("_id") @db.ObjectId
-  subpartId       String        @db.ObjectId
-  subpartUserAns  String[]
-  UserResponse    UserResponse? @relation(fields: [userResponseId], references: [id])
-  userResponseId  String?       @db.ObjectId
+  id             String        @id @default(auto()) @map("_id") @db.ObjectId
+  subpartId      String        @db.ObjectId
+  subpartUserAns String[]
```

Comment:
can I ask what subpartUserAns is used for?

Reply:
This isn't what I've done; the code formatter added some space.